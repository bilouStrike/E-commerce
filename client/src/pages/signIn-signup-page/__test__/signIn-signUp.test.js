import { shallow } from 'enzyme';
import React from 'react';
import SignInAndSignUpPage from '../signIn-signUp.component';
import toJson from 'enzyme-to-json';

it('Expect to render sign in and sign up page', ()=>{
    expect(toJson(shallow(<SignInAndSignUpPage/>))).toMatchSnapshot();
});