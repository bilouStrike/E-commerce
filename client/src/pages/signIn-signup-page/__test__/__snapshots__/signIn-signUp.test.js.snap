// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect to render sign in and sign up page 1`] = `
<div
  className="sign-in-and-sign-up"
>
  <Connect(SignIn) />
  <Connect(SignUp) />
</div>
`;
